support-frontend/assets/helpers/productCatalog.tsx

```diff
@@ -1,12 +1,15 @@
+import { OfferBook } from 'components/offer/offer';
+
 export const productCatalog = window.guardian.productCatalog;
 
-export const productCatalogDescription = {
+export const productCatalogDescExclOffers = {
 	SupporterPlusWithGuardianWeekly: {
 		label: 'Digital + print',
 		benefitsSummary: [
 			'The rewards from ',
 			{ strong: true, copy: 'All-access digital' },
 		],
+		offersSummary: undefined,
```

Comment:
Is this necessary?

Reply:
Removed this `offersSummary: undefined,` and `offers: [],`, required adding the associated type in `threeTierLanding.getCardData(productDescription)`, done.